pkg/controller/masteruserrecord/sync.go

```diff
@@ -220,6 +224,27 @@ func (s *Synchronizer) alignReadiness() bool {
 			return false
 		}
 	}
+
+	if condition.IsTrue(s.record.Status.Conditions, toolchainv1alpha1.ConditionReady) && !notificationCreated {
```

Comment:
Instead of that you should do the following after the line 248:
```
	s.record.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(s.record.Status.Conditions, toBeProvisioned())
	if condition.IsNotTrue(s.record.Status.Conditions, toolchainv1alpha1.MasterUserRecordUserProvisionedNotificationCreated) {
		// TODO Create Notification in host. Not in member!
		...
		s.record.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(s.record.Status.Conditions, NotificationCreated())
	}

	return true.

...
// IsNotTrue returns true if the condition with the given condition type has Unknown or False status
func IsNotTrue(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType) bool {
	condition, found := FindConditionByType(conditions, conditionType)
	return !found || condition.Status != apiv1.ConditionTrue
}
```

`IsNotTrue()` function worth moving to common. Next to `IsTrue()`.

Reply:
@alexeykazakov ohhh! good idea! okay, i'll make that change

Reply:
@alexeykazakov just wondering - the resource here is being created in host right? i see the TODO comment and got a bit confused. i think i've addressed this comment here: 912606199bdd2edef96701b3e4b7ba48d55a6e60

Reply:
His comment is related to this line:
https://github.com/codeready-toolchain/host-operator/pull/203/files/fdea53c1e7e61a5a730a3d53caf1a354770af8a0#diff-5d9536e84d383cec9fa71e746d09dc7cR240
you should not create the Notification CR in member - you use memberCluster there.

Reply:
ohhh okay, i think it's fixed here: 6a278695be3aecb4ed2e8ba27cab0e92ef420ac4